igc_lib.py

```diff
@@ -214,22 +226,31 @@ def to_B_record(self):
                   + (extras))
 
 class Thermal:
+    """Stores information about a single thermal in a flight.
```

Comment:
I see that your branch is not synced (i.e. there are changes that are already in xiadz/igc_lib master), could you try `git pull https://github.com/xiadz/igc_lib.git` in your local git repo? Probably some conflicts will arise.